src/get-started/create-service.md

```diff
@@ -326,36 +299,53 @@ Now, just as we did with the `Wallet` structure above, we need to describe the
 same transactions in Rust:
 
 ```rust
-#[derive(Serialize, Deserialize, Clone, Debug, ProtobufConvert)]
-#[exonum(pb = "proto::cryptocurrency::TxCreateWallet")]
+#[derive(Serialize, Deserialize, Clone, Debug, ProtobufConvert, BinaryValue)]
+#[protobuf_convert(source = "proto::TxCreateWallet")]
 pub struct TxCreateWallet {
     pub name: String,
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug, ProtobufConvert)]
-#[exonum(pb = "proto::cryptocurrency::TxTransfer")]
+#[derive(Serialize, Deserialize, Clone, Debug, ProtobufConvert, BinaryValue)]
+#[protobuf_convert(source = "proto::TxTransfer")]
 pub struct TxTransfer {
     pub to: PublicKey,
     pub amount: u64,
     pub seed: u64,
 }
 ```
 
-Service transactions are defined through the enum with the derive of the
-`TransactionSet`
-that automatically assigns transaction IDs based on their declaration order
-starting from `0`:
+To make service support transactions defined above, we need to declare service
+*interface*. Service interface is basically a trait with methods that correspond
+to the transactions processing logic. In our case interface will look as follows:
 
 ```rust
-#[derive(Serialize, Deserialize, Clone, Debug, TransactionSet)]
-pub enum CurrencyTransactions {
-    /// Create wallet transaction.
-    CreateWallet(TxCreateWallet),
-    /// Transfer tokens transaction.
-    Transfer(TxTransfer),
+/// Cryptocurrency service transactions.
+#[exonum_interface]
+pub trait CryptocurrencyInterface<Ctx> {
+    /// Output of the methods in this interface.
+    type Output;
+
+    /// Creates wallet with the given `name`.
+    #[interface_method(id = 0)]
+    fn create_wallet(&self, ctx: Ctx, arg: TxCreateWallet) -> Self::Output;
+    /// Transfers `amount` of the currency from one wallet to another.
+    #[interface_method(id = 1)]
+    fn transfer(&self, ctx: Ctx, arg: TxTransfer) -> Self::Output;
 }
 ```
 
+`exonum_interface` macro generates a glue to dispatch transactions and
+deserialize their payload within service. `interface_method` macro assigns
```

Comment:
```suggestion
deserialize its payload within service. `interface_method` macro assigns
```

Reply:
I think "their" is more suitable here. Native speakers commonly use "their" to the inanimate objects.